src/swoole/src/LaravelRunner.php

```diff
@@ -54,6 +54,8 @@ public function handle(Request $request, Response $response): void
                 $response->header($name, $value);
             }
         }
+
+        $response->status($sfResponse->getStatusCode());
```

Comment:
It's a little funny because if you look into the source code of the Swoole PHP extension you can clearly see they expect "int", not "string".
```c
struct Response {
    // ...
    int status;
    char *reason;
    // ...
}
```

and in the `status()` method definition, again "int":
```c
static PHP_METHOD(swoole_http_response, status) {
    zend_long http_status;
    char *reason = nullptr;
    size_t reason_len = 0;

    ZEND_PARSE_PARAMETERS_START(1, 2)
    Z_PARAM_LONG(http_status)
    Z_PARAM_OPTIONAL
    Z_PARAM_STRING(reason, reason_len)
    ZEND_PARSE_PARAMETERS_END_EX(RETURN_FALSE);
    // ...
}
```

I guess there's a problem with the declaration of this method:
```c
ZEND_BEGIN_ARG_INFO_EX(arginfo_swoole_http_response_status, 0, 0, 1)
    ZEND_ARG_INFO(0, http_code)
    ZEND_ARG_INFO(0, reason)
ZEND_END_ARG_INFO()
```

There're no specifications about the typehints `ZEND_ARG_TYPE_INFO()`.

Reply:
I guess the swoole/ide-helper package is wrong.